components/modules/NewReleases/index.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module '*.jpg';
+declare module '*.gif';
+declare module '*.png';
```

Comment:
what does this file ?

Reply:
when importing static images, i was getting some error that's why i added that file to help me clear the errors